controllers/utils/multicloud_util.go

```diff
@@ -98,13 +98,17 @@ var EnsureInstallPlanIsApproved = func(
 		clusterGroupUpgrade, subscription.Status.InstallPlanRef.Kind, subscription.Status.InstallPlanRef.Name)
 	multiCloudLog.Info("[EnsureInstallPlanIsApproved] Create MCV for InstallPlan", "InstallPlan",
 		subscription.Status.InstallPlanRef.Name, "ns", clusterName)
+	safeName, ok := clusterGroupUpgrade.Status.SafeResourceNames[mcvForInstallPlanName]
```

Comment:
These 3 lines are used throughout the code, though with different params for GetSafeResourceName. Wondering if we could make a function to hold these lines and return the safeName.

Reply:
Done